### DISCUSSION TITLE
Failover setup

### DISCUSSION THREAD

**Discussion by MarcS1975** [docker-mailserver/docker-mailserver#3754] [Q&A]
I am running a basic mailserver and would like to have a 2nd server with the identical mailserver data on it which I can use in case server 1 goes down. I can do the domain re-direction manually but I do need the 2 servers to be in sync. Which folders do I need to rsync?

1) email data -> I can rsync .../var/mail and .../var/mail-state and .../var/log/mail
2) DMS user configuration that changed after the initial start (users, pwds, fail2ban jails etc) **-> where is that stored?**

Is there anything else to replicate?

**Comment by casperklein** [comment]
> Which folders do I need to rsync?

Sync all your mounted volumes and you should be fine.

> DMS user configuration that changed after the initial start (users, pwds, fail2ban jails etc) -> where is that stored?

In your mounted `config` directory.
